src/index.tsx

```diff
@@ -24,18 +24,18 @@
 
  */
 
-import React from "react"
-import * as ReactDOM from "react-dom"
-import { Extension } from "./components/Extension"
-import { ExtensionProvider } from "@looker/extension-sdk-react"
+import React from 'react'
+import * as ReactDOM from 'react-dom'
+import { ExtensionProvider40 } from '@looker/extension-sdk-react'
+import { Extension } from './components/Extension'
```

Comment:
This is now a locally defined `Extension`?

Reply:
It was always there. I think the linter reorganized the imports